101-backup-summary-report/azuredeploy.json

```diff
@@ -0,0 +1,624 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "logicAppName": {
+      "type": "String",
+      "metadata": {
+        "description": "Name of the Logic App to be created."
+      }
+    },
+    "location": {
+      "type": "String",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Azure region in which the Logic App should be created."
+      }
+    },
+    "logicAppWorkspace": {
+      "type": "String",
+      "metadata": {
+        "description": "Name of the Log Analytics workspace that the Logic App should connect to. Note that the Logic App connects to a single workspace, but can query data across multiple workspaces. Use the 'Workspaces To Query' parameter to specify the list of workspaces that the Logic App should query data from."
+      }
+    },
+    "workspaceResourceGroupId": {
+      "type": "string",
+      "metadata": {
+        "description": "Resource Group id of the Log Analytics workspace that the Logic App should connect to. Should be a string of the format '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}'."
```

Comment:
@bmoore-msft, this resourcegroupId is for the existing log analytics workspace. And it can be from any other subscription and ResourceGroup. Hence if we have to call resourceId function, we need two inputs - subscriptionId and resourceGroup.

Having single input to provide ResourceGroupID URL is not recommended?

Reply:
Have split into two params.